fendermint/app/src/app.rs

```diff
@@ -102,6 +103,9 @@ where
     ///
     /// Only loaded once during genesis; later comes from the [`StateTree`].
     builtin_actors_bundle: PathBuf,
+    /// Manages snapshot taking for the blockchain. If None, it will not trigger snapshot of the
```

Comment:
```suggestion
    /// Manages the creation of periodic snapshots for the network. If None, it will not trigger the creation of snapshots for the
```